2023-09-12 10:03:38,009 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-12 10:03:42,957 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-12 10:03:49,836 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-12 10:03:53,725 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-12 10:03:53,982 | ERROR | OMAudit | user=hadoop | ip=10.9.0.17 | op=READ_VOLUME {volume=96879-volume-for-prepare} | ret=FAILURE | VOLUME_NOT_FOUND org.apache.hadoop.ozone.om.exceptions.OMException: Volume 96879-volume-for-prepare is not found
	at org.apache.hadoop.ozone.om.VolumeManagerImpl.getVolumeInfo(VolumeManagerImpl.java:61)
	at org.apache.hadoop.ozone.om.OzoneManager.getVolumeInfo(OzoneManager.java:2610) 
2023-09-12 10:03:57,572 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-12 10:03:57,881 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {prevKey=, prefix=, maxKeys=1000, username=hadoop} | ret=SUCCESS |  
2023-09-12 10:03:58,321 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {prevKey=vol1, prefix=, maxKeys=1000, username=hadoop} | ret=SUCCESS |  
